hammer/tech/stackup.py

```diff
@@ -406,6 +406,22 @@ def get_metals_below_layer(self, name: str) -> List[Metal]:
         except StopIteration:
             raise ValueError("Metal named %s is not defined in stackup %s" % (name, self.name))
 
+    def get_metals_incl_layer(self, name: str) -> List[Metal]:
+        """
+        Get all the metals including the specified metal layer.
+
+        :param index: Index of the metal layer
+        :return: A list of metal layer objects
+        """
+        try:
+            index = next(m.index for m in self.metals if m.name == name)
+            if index > 1:
```

Comment:
Seems unnecessary to check that the list has to have length > 1.

Reply:
Fixed. Though function not used, leaving in because it could be useful.